src/modules/admin/admin.controller.js

```diff
@@ -0,0 +1,70 @@
+const Admin = require("./admin.model.js");
+
+// check admin exsists or not for given email
+const checkAdminExsists = async (email) => {
+  try {
+    const adminExsists = await Admin.findOne({ where: { email: email } });
```

Comment:
vai exist banan ta sonsodhon koren

Reply:
Ops .. typo .. thanks vai .. next time I will be more careful.